Documentation/UserManual/units.md

```diff
@@ -74,10 +74,10 @@ The order of dimensional exponent units is `(m, kg, s, A, K, mol, cd)`.
 | IFCSECTIONMODULUSMEASURE                      | Section modulus                         |              |             | m3              | (3, 0, 0, 0, 0, 0, 0)   | IfcDerivedUnitEnum.SECTIONMODULUSUNIT                      |                                                    |
 | IFCSHEARMODULUSMEASURE                        | Shear modulus                           |              |             | N / m2          | (-1, 1, -2, 0, 0, 0, 0) | IfcDerivedUnitEnum.SHEARMODULUSUNIT                        |                                                    |
 | IFCSOLIDANGLEMEASURE                          | Solid angle                             | steradian    | sr          | sr              | (0, 0, 0, 0, 0, 0, 0)   | IfcUnitEnum.SOLIDANGLEUNIT                                 | IfcSIUnitName.STERADIAN                            |
-| IFCSOUNDPOWERLEVELMEASURE                     | Sound power level                       | decibel      | db          | db              | (0, 0, 0, 0, 0, 0, 0)   | IfcDerivedUnitEnum.SOUNDPOWERLEVELUNIT                     |                                                    |
-| IFCSOUNDPOWERMEASURE                          | Sound power                             | decibel      | db          | db              | (0, 0, 0, 0, 0, 0, 0)   | IfcDerivedUnitEnum.SOUNDPOWERUNIT                          |                                                    |
-| IFCSOUNDPRESSURELEVELMEASURE                  | Sound pressure level                    | decibel      | db          | db              | (0, 0, 0, 0, 0, 0, 0)   | IfcDerivedUnitEnum.SOUNDPRESSURELEVELUNIT                  |                                                    |
-| IFCSOUNDPRESSUREMEASURE                       | Sound pressure                          | decibel      | db          | db              | (0, 0, 0, 0, 0, 0, 0)   | IfcDerivedUnitEnum.SOUNDPRESSUREUNIT                       |                                                    |
+| IFCSOUNDPOWERLEVELMEASURE                     | Sound power level (log ref. 1e-12 W)    | decibel SWL  | db(SWL)     | db              | (0, 0, 0, 0, 0, 0, 0)   | IfcDerivedUnitEnum.SOUNDPOWERLEVELUNIT                     |                                                    |
```

Comment:
Can't we just use 'decibel' unit, without the SPL/SWL? Whether it's the perceived noise (SPL) or source specs (SWL) it's defined in a property.

Reply:
Your're right, I was attempting to ensure that the identification was unique, but it can be resolved by the measure in context.